README.md

```diff
@@ -79,10 +79,31 @@ Our staging and production environments are hosted by AWS. We would deploy to pr
 
 ## Testing
 
-To run tests:
+### Run the tests
 ```sh
 $ make test
 ```
+### Agreed Testing Approach
+- Use nUnit and Moq
+- Always follow a TDD approach
+- Tests should be independent of each other
+- Gateway tests should test against a real database
+- Test coverage should never go down
+- All use cases should be covered by E2E tests
+- Optimise when test run speed starts to hinder development
+- Unit tests and E2E tests should run in CI
+- Test database schemas should match up with production database schema
+- Have integration tests which test from the PostgreSQL database to API Gateway
+
+## Data Migrations
+### A good data migration
+- 99.99999% available
```

Comment:
I thought we said 5 nines i.e 99.999% availability. Has this changed?

Reply:
I've deleted this line now due to Bens comment